lib/config/fileconf.go

```diff
@@ -991,6 +991,15 @@ type SSH struct {
 	// X11 is used to configure X11 forwarding settings
 	X11 *X11 `yaml:"x11,omitempty"`
 
+	// MaybeFileCopying enables or disables remote file operations via SCP/SFTP.
+	// We're using a pointer-to-bool here because the system default is to allow
+	// TCP forwarding, we need to distinguish between an unset value and a false
```

Comment:
What is TCP forwarding and how does it related to SFTP?

Reply:
Oops, copy-pasta mistake; done